app/src/main/java/com/example/henzoshimada/feeltrip/ElasticSearchController.java

```diff
@@ -241,6 +242,110 @@ public GetMoodTask(String filterBy){
         }
     }
 
+    public static class AddUserTask extends AsyncTask<User, Void, Void> { //TODO: What if username already exists within the database? We need usernames to be UNIQUE for proper sorting.
+
+        @Override
+        protected Void doInBackground(User ... users ) {
```

Comment:
what are we doing in background? Please add comments or change for a more descriptive name